frontend/src/pages/Home.tsx

```diff
@@ -51,32 +54,30 @@ export function Home() {
   }, [filters]);
 
   return (
-    <Page>
-      <Helmet>
-        <title>Home | USHS Housing Portal</title>
-      </Helmet>
-      <NavBar page="Home" />
-      <HomePageLayout>
-        <FitlerPanel></FitlerPanel>
-        <FilterPadding />
-        <div>
-          <FilterDropdown
-            value={filters}
-            refreshUnits={(filterParams) => {
-              filterParams.approved = filters.approved;
-              setFilters(filterParams);
-            }}
-          ></FilterDropdown>
-          <UnitCardGrid
-            units={units}
-            refreshUnits={(approved) => {
-              const newFilters = { ...filters, approved };
-              fetchUnits(newFilters);
-              setFilters(newFilters);
-            }}
-          />
-        </div>
-      </HomePageLayout>
-    </Page>
+    <FiltersContext.Provider value={{ filters, setFilters }}>
```

Comment:
It also looks like the filter context stuff might be broken. When I click "Back" from the Unit Details page, the filter criteria get reset to their default values.

Reply:
Ah, I did some digging and it looks like the `FilterContext.Provider` got removed in [this commit](https://github.com/TritonSE/USHS-Housing-Portal/commit/02ad2f7d90b5c11bef3d7b5bbc894b013d204660#diff-9f4fbb52c544c388744cd765e62f81f3049a5d32948140308a6a623ee4a01005).